site/src/components/banner.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Banner
+description: The Banner component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D1002%253A7106%26t%3DWU64bU98uaT1nU7A-1
+classType: Component
+classPrefixes:
+  - banner
+---
+
+# Banner
+
+Banner component allows adding a full width notification block to the page to notify user about something that’s not directly related to the page content.
```

Comment:
To decrease confusion, I think we should probably just discuss the purpose without using the word notification

Reply:
Updated